hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -802,20 +802,12 @@ protected void deleteRootDir() throws IOException, InterruptedException {
       return;
     }
     deleteRootRecursively(fileStatuses);
-
-    // Waiting for double buffer flush before calling listStatus() again
-    // seem to have mitigated the flakiness in cleanup(), but at the cost of
-    // almost doubling the test run time. M1 154s->283s (all 4 sets of params)
-    cluster.getOzoneManager().awaitDoubleBufferFlush();
```

Comment:
I'm not sure the removal of this line belongs to this PR?

#5244 still looks required.

Reply:
Yes @smengcl #5244 needs anyway. I think while resolving conflict, it got removed.

Reply:
Ok. 5 out of 9 commits merged to master branch after this PR starts to have flaky `integration (filesystem)` again as a result of this removal. Need to merge #5244 asap.

https://github.com/apache/ozone/actions/runs/6100351514/job/16554482439
https://github.com/apache/ozone/actions/runs/6105400581/job/16568864195
https://github.com/apache/ozone/actions/runs/6107352731/job/16574193577
https://github.com/apache/ozone/actions/runs/6110706683/job/16584346159
https://github.com/apache/ozone/actions/runs/6114248488/job/16595437359